src/checks/addMilestoneLabelsToPRs.test.ts

```diff
@@ -93,4 +93,15 @@ describe(addMilestoneLabelsToPRs, () => {
       labels: ["For Milestone Bug"],
     })
   })
+  it("Doesn't do anything for closed PRs", async () => {
+    const { mockAPI, api } = createMockGitHubClient()
+    mockGetRelatedIssues.mockResolvedValue([{ assignees: [] }])
+
+    const pr = getPRFixture("closed")
+    pr.pull_request.body = `fixes #1123`
+
+    await addMilestoneLabelsToPRs(api, pr, getFakeLogger())
+
+    expect(mockAPI.issues.addLabels).not.toHaveBeenCalled()
+  })
```

Comment:
This test did fail 👍🏻 

![Screen Shot 2020-11-20 at 4 31 24 PM](https://user-images.githubusercontent.com/49038/99852104-47fae300-2b4e-11eb-9789-42e13bd10974.png)